### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by davehunt**
* Removed shebangs
 * Removed CONTRIBUTORS
 * Updated and cleaned up requirements.txt
 * Cleaned up .gitignore

Adhoc: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozorg.adhoc/42/

### CODE REVIEW

**Comment by davehunt** [comment]
@bobsilverberg just a small pull to clean up ahead of further changes, mind reviewing it?

File: requirements.txt [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-certifi==0.0.8
-chardet==2.1.1
-execnet==1.1
-py==1.4.26
-pytest==2.7.0
+beautifulsoup4==4.4.1
+pytest==2.7.3
 pytest-mozwebqa
-requests==2.6.0
-selenium
```
Comment (jgmize) [inline requirements.txt]: why is selenium no longer required?

Comment (bobsilverberg) [inline requirements.txt]: `pytest-mozwebqa` already requires `selenium`, so we don't actually need it here, although I recall @davehunt saying something recently about wanting to keep it anyway, so I am surprised by its removal as well.

**Comment by bobsilverberg** [comment]
@davehunt lgtm, r+, other than my agreement with the question from @jgmize

File: requirements.txt (hunk 1 above)
Comment (davehunt) [inline requirements.txt]: I was unclear when previously agreeing with @bobsilverberg's suggesting of removing selenium. I think we only need to include it if we were pinning to a specific version, as pytest-mozwebqa depends on selenium.

**Pull request merged**